components/builder-api/doc/api.raml

```diff
@@ -241,6 +241,23 @@ securitySchemes:
                                 "id": "73089155726360582",
                                 "flags": 0
                             }
+/chef-events-feed:
```

Comment:
I'd suggest naming this endpoint something a bit more RESTful. A plural representation of each object returned might be named something like `/community-events`